composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "perftools/xhgui-collector",
+    "name": "GrzegorzDrozd/xhgui-collector",
```

Comment:
This shouldn't change.

Reply:
Yes, I know. I will revert it later when I am done with testing - need that to install package into application that I use for testing.